operator/pkg/types/installs.go

```diff
@@ -26,16 +30,20 @@ type ChartFlags struct {
 // ImageSpec defines OCI Image specifications.
 type ImageSpec struct {
 	// Repo defines the Image repo
+	// +kubebuilder:validation:Optional
 	Repo string `json:"repo"`
 
 	// Name defines the Image name
+	// +kubebuilder:validation:Optional
 	Name string `json:"name"`
 
 	// Ref is either a sha value, tag or version
+	// +kubebuilder:validation:Optional
 	Ref string `json:"ref"`
 
 	// Type defines the chart as "oci-ref"
 	// +kubebuilder:validation:Enum=helm-chart;oci-ref;""
+	// +kubebuilder:validation:Optional
 	Type RefTypeMetadata `json:"type"`
```

Comment:
```suggestion
	Type RefTypeMetadata `json:"type,omitempty"`
```

Reply:
// +kubebuilder:validation:Optional does the same thing